src/Billing/Services/Implementations/ChargeRefundedHandler.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Bit.Billing.Constants;
+using Bit.Core.Entities;
+using Bit.Core.Enums;
+using Bit.Core.Repositories;
+using Microsoft.AspNetCore.Mvc;
+using Event = Stripe.Event;
+
+namespace Bit.Billing.Services.Implementations;
+
+public class ChargeRefundedHandler : StripeWebhookHandler
+{
+    private readonly IStripeEventService _stripeEventService;
+    private readonly ITransactionRepository _transactionRepository;
+    private readonly ILogger<ChargeSucceededHandler> _logger;
+    public ChargeRefundedHandler(IStripeEventService stripeEventService,
+        ITransactionRepository transactionRepository,
+        ILogger<ChargeSucceededHandler> logger)
+    {
+        _stripeEventService = stripeEventService;
+        _transactionRepository = transactionRepository;
+        _logger = logger;
+    }
+    protected override bool CanHandle(Event parsedEvent)
+    {
+        return parsedEvent.Type.Equals(HandledStripeWebhook.ChargeSucceeded);
+    }
+
+    protected override async Task<IActionResult> ProcessEvent(Event parsedEvent)
```

Comment:
I'm not sure we should be returning `IActionResult` directly from an underlying Handler. I'd assume the controller should receive some sort of Result and determine whether or not it was successful.